### PR TITLE
drop support for python 2.7 and 3.3

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

**Comment by joemoe** [comment]
This pull request fixes 1 alert - [view on lgtm.com](https://lgtm.com/projects/g/crate/crash/rev/pr-28d72666b4ff37019a18f931696274661828de79)

**fixed alerts:**

* 1 for 'input' function used

---

*Comment posted by [lgtm.com](https://lgtm.com)*

**Comment by joemoe** [comment]
This pull request fixes 1 alert - [view on lgtm.com](https://lgtm.com/projects/g/crate/crash/rev/pr-8bd37cd3f31cb9a93bd39b08b4adc095efd24d0a)

**fixed alerts:**

* 1 for 'input' function used

---

*Comment posted by [lgtm.com](https://lgtm.com)*

File: README.rst [hunk 1]
```diff
@@ -59,6 +60,10 @@ To update, run::
 
     $ pip install -U crash
 
+If you use Python 2.7 or 3.3 with a recent version of pip_, it will install
+only version ``0.23.x`` by default, because newer versions of this package are
+not compatible with Python 2.7/3.3 any more.
+
```
Comment (mfussenegger) [inline README.rst]: Further below mentions that the `crash_standalone_latest` works with py2.7. Did we ensure we're keeping the standalone at `0.23`? If not, can we freeze one and link it as well as py27-crash-standalone or something like that?

Comment (chaudum) [reply README.rst]: https://cdn.crate.io/downloads/releases/ keeps all versions of crash standalone.
I will update the README accordingly.

**Comment by joemoe** [comment]
This pull request fixes 1 alert - [view on lgtm.com](https://lgtm.com/projects/g/crate/crash/rev/pr-9687628e7fc7ee46a0bc8930a84ec1676c0d45aa)

**fixed alerts:**

* 1 for 'input' function used

---

*Comment posted by [lgtm.com](https://lgtm.com)*

File: README.rst (hunk 1 above)
Comment (chaudum) [reply README.rst]: @mfussenegger updated the readme with instructions what version to install for legacy python versions.

**Comment by joemoe** [comment]
This pull request fixes 1 alert - [view on lgtm.com](https://lgtm.com/projects/g/crate/crash/rev/pr-a4e312e29613316b0d086829f36ccc7b3a4e85f4)

**fixed alerts:**

* 1 for 'input' function used

---

*Comment posted by [lgtm.com](https://lgtm.com)*

**Pull request merged**